modules/evm/src/runner/state.rs

```diff
@@ -873,15 +873,6 @@ impl<'config, 'precompiles, S: StackState<'config>, P: PrecompileSet> StackExecu
 		self.state.metadata_mut().access_address(caller);
 		self.state.metadata_mut().access_address(address);
 
-		event!(Create {
-			caller,
-			address,
-			scheme,
-			value,
-			init_code: &init_code,
-			target_gas
-		});
-
```

Comment:
We plan to implement [debug_traceTransaction](#1764), these events may be useful, it is best to keep it

Reply:
it is moved on create method. I believe its the correct place to be. It has enter and exit. Tracing is implemented on chopsticks https://github.com/AcalaNetwork/chopsticks/pull/673

Reply:
First enter event is `transact_create` which forwards the call to `create_inner` and then `emit_exit`
Second is `fn create(` which is called during execution. It will emit create and exit event. Same logic for call method